packages/blade/package.json

```
@@ -247,15 +255,15 @@
     "files": [
       {
         "friendlyName": "Web Components",
-        "path": "./build/components/index.web.js"
+        "path": "./build/components/index.production.web.js"
       },
       {
         "friendlyName": "React Native Components",
         "path": "./build/components/index.native.js"
       },
       {
         "friendlyName": "Web Tokens",
-        "path": "./build/tokens/index.web.js"
+        "path": "./build/tokens/index.development.web.js"
```

Shouldn't this be production?
```suggestion
        "path": "./build/tokens/index.production.web.js"
```

---

Yes.